packages/vertexai/src/types/requests.ts

```diff
@@ -114,6 +114,18 @@ export interface StartChatParams extends BaseParams {
  */
 export interface CountTokensRequest {
   contents: Content[];
+  /**
+   * Instructions that direct the model to behave a certain way.
+   */
+  systemInstruction?: string | Part | Content;
+  /**
+   * <code>{@link Tool}</code> configuration.
+   */
+  tools?: Tool[];
+  /**
+   * Configuration options used for content-related requests.
```

Comment:
hmmm... this description doesn't seem quite right.  Let's riff on something like this?

Configuration with parameter values that control how the model generates a response.

Reply:
Sounds good to me, I applied your suggestion

Reply:
> hmmm... this description doesn't seem quite right.

This comment was copied from the current [devsite docs for `GenerationConfig`](https://firebase.google.com/docs/reference/js/vertexai.generationconfig). Do you think this should also be updated in the `GenerationConfig` interface?


> Configuration with parameter values that control how the model generates a response.

In a JS context, parameters specifically refer to function inputs, while `generationConfig` is an object with properties. I would suggest "Object containing properties that control how the model generates a response", but that seems a bit verbose to me.

Reply:
"Configuration options that control how the model generates a response." ?

Reply:
> "Configuration options that control how the model generates a response." ?

+1